.github/workflows/verify_changes.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Get changed directories
         id: get-changeddir
         env:
-          TYPE: ${{ github.event_name }}
-          REF_NAME: ${{ github.ref_name }}
-          BASE: ${{ github.base_ref }}
+          TYPE: ${GITHUB_EVENT_NAME}
```

Comment:
I'm slightly confused by this versus `.github/workflows/changelog-checker.yml` -- why are the substitutions OK there but not here?